pycon/finaid/views.py

```diff
@@ -425,3 +425,33 @@ def get_value(name, object):
         writer.writerow(data)
 
     return response
+
+
+@login_required
+def receipt_form(request):
```

Comment:
This method is a view, not a form, so it'd be better to give it a name that reflects the behavior this view is implementing, like `receipt_upload` or `receipt_add`.